indigo_app/templates/place/works.html

```diff
@@ -161,6 +161,14 @@
             </div>
           </div>
         {% endfor %}
+
+        {% if perms.indigo_api.bulk_export_work %}
```

Comment:
As discussed, let's move this into the settings view and use the settings permissions to guard it.